cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -326,6 +326,28 @@ def test_host_refresh_relationships(provider, setup_provider):
     wait_for_relationship_refresh(provider)
 
 
+@pytest.mark.rhv3
+@pytest.mark.provider([InfraProvider])
+def test_template_refresh_relationships(appliance, provider, setup_provider):
+    """ Test that host refresh doesn't fail
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Infra
+        caseimportance: high
+        initialEstimate: 1/8h
+        tags: relationship
+    """
```

Comment:
I think, you should remove ```tags``` and ```caseimportance```(By default it is high).

Reply:
@ganeshhubale thanks for the review!
There's no harm in `caseimportance` - I think it's more clear.
What about the tag? I see it in all similar tests, so why should it be removed here?

Reply:
I talked with @digitronik 
We can put it as it is optional.